listen.js

```diff
@@ -1,18 +1,26 @@
 // const flumeView = require('flumeview-reduce')
 const pull = require('pull-stream')
 const CRUT = require('ssb-crut')
+const { where, and, type, isDecrypted, live: dbLive, toPullStream } = require('ssb-db2/operators')
```

Comment:
these are on `ssb.db.operators` too

Reply:
is it fine if i keep importing them from ssb-db2?